.github/workflows/ci.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
         id: validation
         continue-on-error: true
         run: |
-          pushd examples
+          pushd example
```

Comment:
@cprice404 Does this address your `examples` folder concern for the way we are doing this action?

Reply:
renaming the dir to `example` is fine if we need to do that.  My only request is that the Cargo.toml file in the example dir specifies a version of momento that is pulled from crates.io so that our CI/dependabot can test the published packages for now, but i think you are doing that here.